### PR TITLE
ci: use the microbenchmarks-pool

### PR DESCRIPTION
**Pull request by v1v**
### What

From now on we will use the same CI workers for the benchmarking

### CODE REVIEW

File: scripts/jenkins/run-benchmarks.sh [hunk 1]
```diff
@@ -28,6 +28,10 @@ function setUp() {
     then
         CORE_INDEX=7
         BASE_FREQ="1.9GHz"
+    elif [ "${CPU_MODEL}" == "12th Gen Intel(R) Core(TM) i5-12500 " ]
```
Comment (v1v) [inline scripts/jenkins/run-benchmarks.sh:31]: @eyalkoren , I cherry-picked your changes from https://github.com/elastic/apm-agent-java/pull/2693/commits/60593129312c624941832468e0b1fa60fda91714

**Comment by v1v** [comment]
I assume these errors are unrelated to the nature of this change, since it ran the `Benchmarks` without any trouble

**Comment by jackshirazi** [comment]
I think that one has become a flaky test, I'll re-run and investigate if it stays

File: .ci/load/Jenkinsfile [hunk 2]
```diff
@@ -164,7 +164,7 @@ pipeline {
         stage('execute test') {
             parallel {
                 stage('Load generation') {
-                    agent { label 'metal' }
+                    agent { label 'linux && metal' }
```
Comment (cachedout) [inline .ci/load/Jenkinsfile:167]: Why would this not go into the microbenchmarks pool?

**Review by jackshirazi — APPROVED** [review]

File: .ci/load/Jenkinsfile (hunk 2 above)
Comment (v1v) [reply .ci/load/Jenkinsfile:167]:
```suggestion
                    agent { label 'microbenchmarks-pool' }
```

**Review by jackshirazi — APPROVED** [review]

**Review by cachedout — APPROVED** [review]

**Pull request merged**